### ISSUE TITLE
AndroidAccountStore - LoadEmptyKeyStore - KeyStore was not initialized

### ISSUE THREAD

**Issue by bartdkmediahuis** [xamarin/Xamarin.Auth#239]
# Xamarin.Auth Issue

### Version
- [ ] nuget version = 1.5.0

### Steps to reproduce

1. Hard to reproduce this issue, since we only have exception logs of end-users facing this problem. 

```
Exception: KeyStore was not initialized
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <506a42a3a126462eb26f593d3ae1120c>:0 
  at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualObjectMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <742a6edd960b4fb699cb9ccbd74061bf>:0 
  at Java.Interop.JniPeerMembers+JniInstanceMethods.InvokeNonvirtualObjectMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters) [0x0001f] in <742a6edd960b4fb699cb9ccbd74061bf>:0 
  at Java.Security.KeyStore.Aliases () [0x0000a] in <11c47835c3f94ca5b4ed0336efc2f1d9>:0 
  at Xamarin.Auth.AndroidAccountStore.FindAccountsForServiceAsync (System.String serviceId) [0x00012] in <8f2237247f7844b1a995cb5465cdbfbe>:0 
```
Wondering if this specific code is still required ? (**AndroidAccountStore**)

<summary>Work around Bug https://bugzilla.xamarin.com/show_bug.cgi?id=6766 </summary>
void LoadEmptyKeyStore(char[] password)

**Comment by Macjon** [comment]
Any updates?

**Comment by TamasRusvai** [comment]
Duplicated #223

**Comment by newky2k** [comment]
this has been fixed in 1.6.0.2
